docs/contributing/philosophy.md

```diff
@@ -43,11 +43,11 @@ Given the same input, Semgrep gives the same output.
 Semgrep can run without internet access so developers can write code from airplanes or beaches.
 
 1. **Rules are safe to run no matter where they came from**<br/>
-Rules shouldn’t have the capability to run arbitrary code on your system, only to act as a function that produces a deterministic output message. We may let the user explicitly violate this trust boundary through flags like —dangerously-run-rules.
+Rules shouldn’t have the capability to run arbitrary code on your system, only to act as a function that produces a deterministic output message.
 
 1. **Single-file analysis**<br/>
-To stay fast and limit complexity, we draw a line at crossing file boundaries during analysis. We lose the ability to detect certain complex interprocedural issues, but that’s an explicit tradeoff we make.<br/><br/>
-Our goal is to catch what a senior engineer would catch in code review: Semgrep isn’t designed to find a crazy issue that’s 300 calls from start to finish and evaded the team for 20 years. Instead, it’s designed for enforcing best-practices and automating the code review tasks that an excellent senior engineer would be capable of.<br/><br/>
+To stay fast and limit complexity, we draw a line at crossing file boundaries during analysis. We lose the ability to detect certain complex interprocedural issues, but that’s an explicit tradeoff we make. A blog post by <br/><br/>
+Our goal is to catch what a senior engineer would catch in code review: Semgrep isn’t designed to find a crazy issue that’s 300 calls from start to finish and evaded the team for 20 years. Instead, it’s designed for enforcing best-practices and automating the code review tasks that an excellent senior engineer would be capable of. For a discussion of why expressive creativity is better than a powerful engine, [see this excellent blog post by Devdatta Akhawe](https://devd.me/log/posts/static-analysis/).<br/><br/>
```

Comment:
Not blocking comment (just thinking out loud): I see how the whole file holds together with `<br/>`... I think line breaks are not encouraged, as different browsers may interpret them differently. But <p /> does not seem like a very clean solution either.